packages/salesforcedx-vscode-core/src/context/workspaceContext.ts

```diff
@@ -1,47 +1,29 @@
-import { AuthInfo, Connection } from '@salesforce/core';
-import { join } from 'path';
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Connection } from '@salesforce/core';
+import { OrgInfo, WorkspaceContextUtil } from '@salesforce/salesforcedx-utils-vscode/out/src/context';
 import * as vscode from 'vscode';
 import { setupWorkspaceOrgType } from '.';
-import { SFDX_CONFIG_FILE, SFDX_FOLDER } from '../constants';
-import { nls } from '../messages';
-import { telemetryService } from '../telemetry';
-import { getRootWorkspacePath, OrgAuthInfo } from '../util';
-
-export interface OrgInfo {
-  username?: string;
-  alias?: string;
-}
 
 /**
  * Manages the context of a workspace during a session with an open SFDX project.
  */
 export class WorkspaceContext {
```

Comment:
WorkspaceContext is now basically a wrapper around WorkspaceContextUtil, with org type handling now done in the handleCliConfigChange listener.